automated install/basic-install.sh

```diff
@@ -702,6 +716,32 @@ find_IPv4_information() {
     IPV4_ADDRESS=$(ip -oneline -family inet address show | grep "${IPv4bare}/" | awk '{print $4}' | awk 'END {print}')
 }
 
+confirm_ipv6_only() {
+    # Confirm from user before IPv6 only install,
+    # unless PI_HOLE_IPV6_ONLY environment variable is true,
+
+    if [ "$PIHOLE_IPV6_ONLY" != true ]; then
```

Comment:
What's the purpose of this env. variable?
On a fresh installation, also other dialog will be presented. Why just skip this one?

Reply:
removed